guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -3,7 +3,7 @@
 
 Before installing {SmartProxyServer} packages, you must update all packages that are installed on the base operating system.
 
-ifdef::foreman-el,katello,satellite[]
```

Comment:
Is this only for the link to el7? If so, then you'd need to slightly adjust the order/add an additional "ifdef".

Reply:
For Satellite, I am removing xrefs as there is no EL7. So, kept separate `ifdef` for it, in the end.